wazuh/indexer_stack/wazuh-indexer/cluster/indexer-sts.yaml

```diff
@@ -122,6 +129,14 @@ spec:
               mountPath: /usr/share/wazuh-indexer/opensearch-security/internal_users.yml
               subPath: internal_users.yml
               readOnly: true
+            - name: indexer-conf
```

Comment:
I'm not sure, why you are adding these mounts. So fare `config.yml` and `roles_mapping.yml` aren't referenced anywhere. This results in an error while creating the indexer-sts.

Reply:
@mirwanda thank you very much for keeping an eye. I will fix and return to you.

Reply:
@mirwanda My bad - they were from my custom config as I set up a separate integration with Okta. Now should be fine.

Reply:
Perfect. I can confirm. Thanks!